test/e2e/specs/onboarding/signup__videopress-tailored.ts

```diff
@@ -83,10 +85,10 @@ describe( DataHelper.createSuiteTitle( 'Signup: Tailored VideoPress' ), () => {
 		} );
 
 		it( 'Land in processing', async function () {
-			await page.waitForURL( /.*processing.*/, { timeout: 20 * 1000 } );
+			await page.waitForURL( /.*processing.*/, { timeout: 60 * 1000 } );
```

Comment:
This is a bit out of the scope of this PR but, if possible, let's use user-facing attributes for content validation. Any element/text on page would be better here, since checking the URL alone doesn't ensure the user sees any content: The URL can be awaited correctly, but the page might still be a WSOD.

An alternative could be:

```js
await this.page.getByText( 'some texton the processing page' ).waitFor();
```